packages/adapter-vercel/files/serverless.js

```diff
@@ -35,12 +35,23 @@ export default async (req, res) => {
 
 	const request = await getRequest({ base: `https://${req.headers.host}`, request: req });
 
-	setResponse(
-		res,
-		await server.respond(request, {
-			getClientAddress() {
-				return /** @type {string} */ (request.headers.get('x-forwarded-for'));
-			}
-		})
-	);
+	const response = await server.respond(request, {
+		getClientAddress() {
+			return /** @type {string} */ (request.headers.get('x-forwarded-for'));
+		}
+	});
+
+	if (skew_protection) {
+		response.headers.set(
```

Comment:
Does this get called for "pages" only or also for sub requests like `/api` style functions?

If both you probably want to do this if `Sec-Fetch-Dest` is `document`

Reply:
It gets called for server `load` functions and `/api` style functions too, though i figured it would be harmless since those requests would be routed to the same deployment anyway, making the cookie a no-op

I didn't want to introduce sec-fetch-dest because Safari doesn't implement it, but maybe it's just a harmless optimisation?

Reply:
Ah, you're right that it doesn't hurt on sub requests. AFAIK Safari shipped those headers a while ago. In fact, Vercel will only accept the cookie for sub requests based on the same mechanism

Reply:
ah you're right, they shipped it in 16.4. won't hurt to serve fewer bytes, will make this change